tomls/pools/spartan-council.toml

```diff
@@ -6,7 +6,6 @@
 
 [invoke.createScPool]
 target = ["system.CoreProxy"]
-from = "<%= settings.deployer %>"
```

Comment:
won't this redeploy pool?
also do we have permissions added to deployer in some places (permissions.toml)?

Reply:
does not